src/Components/Inventory/Inventory.js

```diff
@@ -1,14 +1,16 @@
 import React from 'react'; 
 import WeaponsCard from '../WeaponsCard/WeaponsCard';
 
-export default function Inventory({ weapons }) {
+export default function Inventory({ weapons, setWeapon }) {
 
   const allWeapons = weapons.map(weapon => {
     return (
         <WeaponsCard
           name={weapon.name}
```

Comment:
I would have just imported one props and called the props you need with dot notation in the weapons card. But thats personal opinion, this works fine.